addons/zylann.hterrain/shaders/multisplat16.shader

```diff
@@ -25,6 +25,9 @@ uniform bool u_depth_blending = true;
 uniform float u_globalmap_blend_start;
 uniform float u_globalmap_blend_distance;
 uniform bool u_tile_reduction = false;
+uniform float low_edge = 0.1;
+uniform float smoothness = 0.2;
+uniform float height_boost = 0.1;
```

Comment:
Should respect naming convention, uniforms start with `u_` (I wish I could use `_` only but Godot reserves that).
Also I have no idea what these do just from the name. I see they all affect how textures are blended but their name alone doesnt reflect that either.